apps/website/src/components/ui/InstallButton.tsx

```diff
@@ -1,33 +1,83 @@
 'use client';
 
-import { Copy, CopyCheck } from 'lucide-react';
+import { ChevronDown, Copy, CopyCheck } from 'lucide-react';
 import { useEffect, useState } from 'react';
 import { useCopyToClipboard } from 'usehooks-ts';
 
 export function InstallButton({ className = '' }: { readonly className?: string }) {
 	const [interacted, setInteracted] = useState(false);
 	const [copiedText, copyToClipboard] = useCopyToClipboard();
+	const [showDropdown, setShowDropdown] = useState(false);
+
+	const handleCopy = async (command: string) => {
+		setInteracted(true);
+		await copyToClipboard(command);
+		setShowDropdown(false);
+	};
 
 	useEffect(() => {
 		const timer = setTimeout(() => setInteracted(false), 2_000);
 		return () => clearTimeout(timer);
 	}, [interacted]);
 
 	return (
-		<button
-			className={`cursor-copy rounded-md border border-neutral-300 bg-white px-4 py-2 font-mono hover:bg-neutral-200 dark:border-neutral-700 dark:bg-transparent dark:hover:bg-neutral-800 ${className}`}
-			onClick={async () => {
-				setInteracted(true);
-				await copyToClipboard('npm install discord.js');
-			}}
-			type="button"
-		>
-			<span className="font-semibold text-blurple">{'>'}</span> npm install discord.js{' '}
+		<div className={`relative inline-block ${className}`}>
+			<button
+				className="cursor-pointer rounded-md border border-neutral-300 bg-white px-4 py-2 font-mono hover:bg-neutral-200 dark:border-neutral-700 dark:bg-transparent dark:hover:bg-neutral-800"
+				onClick={() => setShowDropdown(!showDropdown)}
+				type="button"
+			>
+				<span className="font-semibold text-blurple">{'>'}</span> npm install discord.js
```

Comment:
This makes it look really weird, `npm` is always shown but dropdown select makes it copy a diff package manager